macOS/DuckDuckGo/Preferences/Model/StartupPreferences.swift

```diff
@@ -35,6 +64,9 @@ struct StartupPreferencesUserDefaultsPersistor: StartupPreferencesPersistor {
     @UserDefaultsWrapper(key: .customHomePageURL, defaultValue: URL.duckDuckGo.absoluteString)
     var customHomePageURL: String
 
+    @UserDefaultsWrapper(key: .startupWindowType, defaultValue: .window)
```

Comment:
Ideally we should stop using `UserDefaults` (and `UserDefaultsWrapper`) and introduce `ThrowingKeyValueStore` to `StartupPreferencesUserDefaultsPersistor`.

See e.g. `AppearancePreferencesUserDefaultsPersistor.isOmnibarVisible`.

Reply:
Sure. Let me change it to use `ThrowingKeyValueStore`